lib/Optimized64/KeccakP-1600-AVX512-plainC.c

```diff
@@ -240,7 +240,7 @@ size_t KeccakP1600_AVX512_12rounds_FastLoop_Absorb(void *state, unsigned int lan
         #undef laneCount
         copyToState(stateAsLanes);
     } else if (laneCount == 17) {
-        // TODO: hardcode `laneCount` for 17?
+        // TODO: this case was untested.
```

Comment:
OK, I tested it on my machine that supports AVX512.

But we should leave a TODO for further optimizations.